docs/utilities/idempotency.md

```diff
@@ -909,6 +910,22 @@ You can create your own persistent store from scratch by inheriting the `BasePer
 
     For example, the `_put_record` method needs to raise an exception if a non-expired record already exists in the data store with a matching key.
 
+### Modifying the Idempotent Repsonse
```

Comment:
Not everyone will want to modify the response. Perhaps `Accessing the Idempotent Response` would be more appropriate.

Reply:
What about `Manipulating the Idempotent Response`?